app/assets/stylesheets/new_styles/_forms.scss

```diff
@@ -4,122 +4,111 @@ input:-webkit-autofill{
   background-image: none !important;
 }
 
-
 form.block-form {
-  display : inline-block;
+  margin: 20px auto;
 
   label {
-    color : $text-dark-grey;
-  }
-
-  input {
-    &:invalid,
-    &:invalid:required,
-    &:invalid:required:focus {
-      -webkit-box-shadow : none;
-      -moz-box-shadow : none;
-      box-shadow : none;
-      box-shadow : none;
-
-      border : none;
-      color : $text-dark-grey;
-    }
+    color: $text-dark-grey;
+    position: absolute;
+    top: -9999px;
+    left: -9999px;
   }
 
   fieldset {
-    @include border-radius(5px);
-
-    -webkit-box-shadow : inset 0 1px 1px rgba(0,0,0,0.2), 0 1px 2px rgba(255,255,255,0.7);
-
-    margin-bottom : 1em;
-
-    background-color : #fff;
-
-    border : 1px solid $border-dark-grey;
-
-    input[type=text],
-    input[type=email],
-    input[type=password] {
-      @include box-shadow(0,0,0,0);
-      @include border-radius(0);
-
-      background : transparent;
-
-      padding : 10px;
-
-      margin-bottom : 0;
-      border : none;
+    margin-bottom: 1em;
+    background-color: #fff;
+    position: relative; /* To correctly place the entypo icon */
+  
+    input { /* Bootstrap reset */
+      &,
+      &:focus, 
+      &:invalid,
+      &:invalid:focus,
+      &:invalid:required,
+      &:invalid:required:focus {
+        border: 1px solid $border-grey;
+        border-bottom: 0px;
+        border-radius: 0px;
+        box-shadow: none;
+        color : $text-dark-grey;
+        margin: 0px;
+      }
     }
 
-  /* mainly bootstrap overrides */
-    .control-group {
-      margin : 0;
-      border-bottom : 1px solid $border-grey;
-
-      .control-label,
-      input[type=text],
-      input[type=password],
-      .field_with_errors label {
-        padding : 10px;
-        margin : 0;
+    .form-control {
+      font-size: 16px;
```

Comment:
I think font-size would be better specified in rem's. As a rule of thumb, take this: 
anything width-related - use percentages
anything height-related, +paddinga +margins - use rems

That alone gives you a quite responsive site. 
In this case, as the `<html>` font-size is set to 100% probably (rems are relative units to the html and body font-size), 16px == 1rem